modules/configuration/pages/secrets.adoc

```diff
@@ -15,14 +16,51 @@ thing:
 [NOTE]
 .Use quotes
 ====
-Note that it would be valid to have `+super_secret: ${SECRET}+` above (without the quotes), but if `SECRET` is unset then the configuration becomes structurally different. Therefore, it's always best to wrap environment variable interpolations with quotes so that when the variable is unset you still have a valid configuration (with an empty string).
+It is valid to have `+super_secret: ${SECRET}+` above (without the quotes), but if `SECRET` is unset then the configuration becomes structurally different. Therefore, it's always best to wrap environment variable interpolations with quotes so that when the variable is unset you still have a valid configuration (with an empty string).
 ====
 
-More information about this syntax can be found on the xref:configuration:interpolation.adoc[interpolation field page].
+For more information about this syntax, see the xref:configuration:interpolation.adoc[interpolation field page].
 
-== Using CLI flags
+== Look up secrets on a remote system at runtime
 
-As an alternative to environment variables, you can set specific fields within a configuration using the CLI flag `--set`, where the syntax is a `<path>=<value>` pair:
+From version 4.39.0, you can use the `rpk connect` CLI flag `--secrets` to look up secrets values on a remote system at runtime. For example, in your secrets management solution. This means that Redpanda Connect resolves interpolations in your configuration at runtime without setting environment variables.
+
+
+For example, you could run the following command to retrieve the value for `"$\{SECRET}"`, when the secret is stored on a Redis server.
+
+```bash
+
+rpk connect run ./config.yaml --secrets redis://secret:redis@example.com
+
+```
+The command tries to load the secret value from the specified URN using the format `scheme://secret:server_address`.
```

Comment:
URN? Should it be URL? I don't see URN spelled out anywhere here. What is it? Noticed there is no glossary in RPCN docs...maybe time to think about one?